x-pack/plugins/ml/server/lib/alerts/register_jobs_monitoring_rule_type.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { KibanaRequest } from 'kibana/server';
+import { ML_ALERT_TYPES } from '../../../common/constants/alerts';
+import { PLUGIN_ID } from '../../../common/constants/app';
+import { MINIMUM_FULL_LICENSE } from '../../../common/license';
+import {
+  anomalyDetectionJobsHealthRuleParams,
+  AnomalyDetectionJobsHealthRuleParams,
+} from '../../routes/schemas/alerting_schema';
+import { RegisterAlertParams } from './register_ml_alerts';
+import {
+  ActionGroup,
+  AlertInstanceContext,
+  AlertInstanceState,
+  AlertTypeState,
+} from '../../../../alerting/common';
+
+export type AnomalyDetectionJobsHealthAlertContext = {
+  jobIds: string[];
+  message: string;
+} & AlertInstanceContext;
+
+export const ANOMALY_DETECTION_JOB_REALTIME_ISSUE = 'anomaly_detection_realtime_issue';
+
+export type AnomalyDetectionJobRealtimeIssue = typeof ANOMALY_DETECTION_JOB_REALTIME_ISSUE;
+
+export const REALTIME_ISSUE_DETECTED: ActionGroup<AnomalyDetectionJobRealtimeIssue> = {
```

Comment:
Are we having one rule type for all the operational issues (real-time and job issues)? If so, all the 'realtimes' here should be switched to 'operational' I think e.g. `Operational issue detected' here:

![image](https://user-images.githubusercontent.com/7405507/126331190-31fd4f0d-10ca-4dbf-b7df-c4a944fb70b3.png)

And if not, then the convention is to hyphenate `real-time`, so variable names and text should be adjusted accordingly.

Reply:
Fixed in https://github.com/elastic/kibana/pull/106084/commits/9c0b3ecfa61edfea97ca0881482ca69af5ebd954